Meadowlark-js/tests/system/crud/SystemTestSetup.ts

```diff
@@ -16,7 +16,7 @@ import { Logger } from '@edfi/meadowlark-utilities';
 const TEST_SIGNING_KEY =
   'zE1mgVCZ01pO5gEioEp/ShcdVMcYjvGMMyE37Tu513WzSgDr4gtMKAFSWrprwXa825Eeu7xj2Ok35KFChE+6oyDF0PuVQdf9YaE9iTTs/nLXDbOJspDvQswYZ7Iq5Mx6lIcixgLbWcK/OckOin608ivPa5aLkJOJXDdl2cgwgf1E9rHWuRs0iYNWoMbqGN2iFH5af3wOF/RusrWJDLLJ9riWeIINeZwzE9cLq75hLBHLpQlcz9enU1zC49B93OTPWPwCqZIBaSqvbsgSULYANoU0JZa+NB5BsgrmlQgldfEPLQj73z09a7yqLAwlJJRYXuJQo6onYQ7RVVHD4PD2Sg==';
 process.env.OAUTH_SIGNING_KEY = TEST_SIGNING_KEY;
-process.env.MEADOWLARK_DATABASE_NAME = 'meadowlark_system_tests';
+process.env.MEADOWLARK_DATABASE_NAME = 'meadowlark';
```

Comment:
This is a workaround to fix the issue when running the System tests alone. When setting up a clean environment, it was depending on the integration tests to create the db in order for it to run. I created a follow up ticket to update this and use TestContainers to isolate the execution